docs/contributor/configuration.md

```diff
@@ -0,0 +1,76 @@
+# Configuration
+
+Learn about the [default configuration](#default-configuration) in Busola and [how to change it](#changing-the-configuration).
+
+## Default Configuration
+
+Busola is delivered with the following default settings:
+
+| Parameter  | Comment                                                                                                                                          | Default Value           |
+| ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------- |
+| `features` | Switches a set of Busola features on and off. Use selectors to configure conditions for the features. To switch them off, set `isEnabled=false`. | `isEnabled=true`        |
```

Comment:
So, do I read it correctly, that the default config in yaml notation is the following?
```yaml
features: "isEnabled=true"
```
literally. Meaning with this config option i would be enabling all of the features, and not being able to cherry-pick the ones I need?